imagebitmap-renderingcontext/bitmaprenderer-as-imagesource.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Canvas's ImageBitmapRenderingContext test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-imagebitmap-rendering-context">
+<script>
+var width = 10;
+var height = 10;
+
+function testCanvas(ctx, x, y, r, g, b, a)
+{
+    var color = ctx.getImageData(x, y, 1, 1).data;
+    assert_array_equals(color, [r, g, b, a]);
+}
+
+function consumeImageBitmap(image, t)
+{
+    var myCanvas = document.createElement('canvas');
+    myCanvas.width = myCanvas.height = 20;
+    var myCtx = myCanvas.getContext('bitmaprenderer');
+    myCtx.transferFromImageBitmap(image);
+
+    return createImageBitmap(myCanvas).then(t.step_func_done(function(imageBitmap) {
+        assert_equals(imageBitmap.width, 20);
```

Comment:
According to the spec imageBitmap should be 10x10, the size of the image that was transferred into myCtx.  What does Firefox do here?